runtime/compiler/build/files/common.mk

```diff
@@ -157,7 +158,7 @@ JIT_PRODUCT_BACKEND_SOURCES+=\
     omr/compiler/optimizer/BackwardUnionBitVectorAnalysis.cpp \
     omr/compiler/optimizer/BitVectorAnalysis.cpp \
     omr/compiler/optimizer/CatchBlockRemover.cpp \
-    omr/compiler/optimizer/CFGSimplifier.cpp \
+    omr/compiler/optimizer/OMRCFGSimplifier.cpp \
     omr/compiler/optimizer/CompactLocals.cpp \
```

Comment:
The travis build failed because this did not include the corresponding cmake change.

[1] https://travis-ci.org/eclipse/openj9/jobs/544364146